js/web/lib/onnxjs/instrument.ts

```diff
@@ -220,6 +227,27 @@ namespace log {
 
     // TODO: we want to support wildcard or regex?
   }
+
+  export function setWithEnv(env: Env): void {
+    let config: Logger.Config = {};
+    if(env.loggingLevel) {
```

Comment:
run 'npm run lint' and 'npm run format' before pushing. I think it's going to complain about some styles.

Reply:
yea thanks, i always forget that part.

Reply:
you can also follow `/js/README.md` to use VSCode with extensions. Extension config is already setup and checked in in the code base.